src/chrome/content/rules/multistre.am.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="multistre.am">
+	<target host="multistre.am" />
```

Comment:
There seems to be a bug with the website where https://www.multistre.am/ redirects to the HTTP version, which is why I didn't.
The www. only redirects to non-www anyway. Are you sure that I should?